### ISSUE TITLE
Old versions of package on pypi

### ISSUE THREAD

**Issue by hellpain** [jazzband/django-configurations#160]
When I open  [django-configurations page on pypi](https://pypi.python.org/pypi/django-configurations) I see only the last version of package and furthermore it is dev version.
Please make available full list of package versions in pypi project page settings.
Good example is [django](https://pypi.python.org/pypi/Django) where I can find all versions.
Thanks!

**Comment by amureki** [comment]
@hellpain if it is urgent, you can use [this link](https://pypi.python.org/simple/django-configurations/) to get all versions list on pypi.

For full list on normal pypi page we should get someone who has access to pypi account and turn on this feature in package settings.

**Comment by hellpain** [comment]
It is not urgent. I could get version 1.0 by modifying url by hands [https://pypi.python.org/pypi/django-configurations/1.0](https://pypi.python.org/pypi/django-configurations/1.0). But it is not obvious, I have spent about half hour for this and not everybody would think of this.
Also I must say that I needed only .tar.gz file, not wheel package and "pip download" always downloaded wheel.
I am sure it is a good style to show all versions to variety of reasons.

**Comment by jezdez** [comment]
Enabling ever new version to be shown on the PyPI project page is a no-go as it requires a manual step during the release. django-configurations is automatically released from a Git tag and PyPI doesn't project an API to publish all versions retroactively once a new release is done. The good news is that the future version of PyPI will have a nice history view of all project releases, e.g.: https://pypi.io/project/django-configurations/#history

Closing as wontfix.

**Comment by syurchenko** [comment]
I think you misunderstood me.
Release history is not connected to your way of publishing packets.
You just can turn off **Auto-hide old releases** in packet settings on pypi and one time manually make visible old releases.
This settings are available via [https://pypi.python.org/pypi?%3Aaction=pkg_edit&name=django-configurations](https://pypi.python.org/pypi?%3Aaction=pkg_edit&name=django-configurations) for this project when you are logged in on pypi.

**Comment by jezdez** [comment]
Ah, thanks for the clarification @syurchenko, that makes sense. I've disabled the option and showed all versions.
